raco/algebra.py

```diff
@@ -462,6 +478,15 @@ def get_equijoin_condition(col0, col1):
         return expression.EQ(expression.UnnamedAttributeRef(col0),
                              expression.UnnamedAttributeRef(col1))
 
+    def partitioning(self):
+        """ The schemas are mutually exclusive so union the partition attributes"""
+        return RepresentationProperties(hash_partitioned=set.union(self.left.partitioning(),
+                                                                self.right.partition()))
+
+    def scheme(self):
+        """Return the scheme of the result."""
+        return self.left.scheme() + self.right.scheme()
```

Comment:
I'm not seeing why this is correct.  Why is the scheme not just the left?  If this is Symmetric difference, it's still the left.  Is it like a full outer join or something?